CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to awpy
+
+Hi! We're happy to have you here. Thank you in advance for your contribution to awpy.
+
+## Python code
+
+To make sure that awpy code stays readeable and works correctly we are making use of
+a variaty of helpfull tools.
+
+At the time of writing these are:
+
+- [black](https://github.com/psf/black): An uncompromising Python code formatter.
+- [ruff](https://github.com/astral-sh/ruff): An extremely fast Python linter, written in Rust.
+- [pyright](https://github.com/microsoft/pyright): A static type checker for Python.
+- [pylint](https://github.com/pylint-dev/pylint): A static code analyser for Python.
+- [pytest](https://docs.pytest.org/en/7.4.x/): A mature full-featured Python testing tool.
+- [pre-commit](https://pre-commit.com/): A framework for managing and maintaining multi-language pre-commit hooks.
+
+To make sure that you pull request is easy to review and can be merged quickly please
+please install these after cloning the repo.
+
+For that simply run the following commands:
+```shell
+pip install -r tests/requirements.txt
+pre-commit install --install-hooks
+```
+
+After you have made your changes locally run these tools to make sure there are no
```

Comment:
After you have made your changes locally, use these tools to surface bugs or code smells by running the following: